scripts/accumulator-server.py

```diff
@@ -44,8 +44,15 @@
 import atexit
 import string
 import signal
+from OpenSSL import SSL
 
-
+# Set up TLS
+dir = os.path.dirname(argv[0])
+if dir == '':
+    dir = os.curdir
+context = SSL.Context(SSL.SSLv23_METHOD)
+context.use_certificate_file(os.path.join(dir, 'accumulator.crt'))
```

Comment:
As discussed by skype, let's have this hardwired to /tmp or alternatively move it to CLI parameters (in the later case, a refactoring in the arguments consumtion logic is desirable, given that it's getting complex with so many parameters).